compiler/modules/front/stmt.bal

```diff
@@ -215,6 +216,28 @@ class StmtContext {
         return resolveSubsetTypeDesc(self.mod, self.functionDefn, td);
     }
 
+    function addNarrowings(bir:BasicBlock bb, Environment env, Narrowing[] narrowings, Position pos) {
```

Comment:
Why is this a method?

Reply:
It has to be called form expr.bal, I assumed expr.bal shouldn't call functions in stmt.bal (unless methods in context).